.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+name: spell
+on:
+  push:
+    branches:
+    - master
+    paths:
```

Comment:
I guess if you want to filter on paths for this workflow, you would need to include "**.3", "**.33" and "**.1", right?

Reply:
`**.3`  and `**.1` at least, since the `.33` files are just the trick this job makes as an intermediate step